### ISSUE TITLE
The id value of the form should be more specific

### ISSUE THREAD

**Issue by kostasx** [In-tech-gration-Cohort-0x02/Bookaholics#12]
https://github.com/In-tech-gration-Cohort-0x02/Bookaholics/blob/16cda93ad2e19328b6dfdfc072d25f1ff207b627/index.html#L14

The name "form" is way too generic for an id. What is this form all about? Find an id that better describes the form's function and role.

**Comment by leonalkalai** [comment]
Would an id "getBooks" be more suitable for this form that gets book?

**Comment by leonalkalai** [comment]
Would an id "getBooks" be more suitable for this form that gets book?

**Comment by kostasx** [comment]
No, getBooks is a verb and implies action. Something like getBooks should preferrably be reserved for the function handling such actions, e.g. form.addEventListener("submit"). Something like "book-search" should be more appropriate for the form ID.

**Comment by leonalkalai** [comment]
@saligaros can you change the form id to "book-search" please?
